lib/core/src/Cardano/Pool/DB.hs

```diff
@@ -47,6 +47,7 @@ import System.Random
 -- with @atomically@:
 --
 -- Example:
+--
```

Comment:
This code example was unreadable, because it was made part of paragraph. Separating the code block from surrounding text fixes formatting.